docs/faq.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: FAQ
+---
+
+* TOC
+{:toc}
+
+## What browser does best on web-platform-tests?
+
+This is a question we've strongly avoided answering: we don't want to
```

Comment:
I think this is written in a way that draws undue attention to our cynicism :)

I think it's possible to answer the same question in a more positive way e.g.

> web-platform tests is intended to help browser vendors make more interoperable implementations. It hasn't been designed to produce metrics that are useful for simple comparisons between browsers, and as such any such comparison based on web-platform-tests results is likely to be misleading. For example, a feature might have an incomplete testsuite, leading to an inflated score, or a testsuite with a large number of tests for edge cases that are believed to have little real-world compatibility impact. Therefore making a valid interpretation of the results requires detailed knowledge of the tests and the feature under test.
>
> By avoiding scoring browsers, we hope to create an environment in which vendors feel able to release their tests that expose bugs and behavioural differences, so making it easier to find and fix such issues before they become site compatibility problems.

Reply:
While the caveats remain valid, nowadays http://wpt.fyi exists and is official.